slackbot/dispatcher.py

```diff
@@ -138,9 +138,14 @@ def loop(self):
         while True:
             events = self._client.rtm_read()
             for event in events:
-                if event.get('type') != 'message':
-                    continue
-                self._on_new_message(event)
+                if event.get('type') == 'message':
```

Comment:
That's fine. Once I've got a little bit of spare time I'll make that change for you :)